src/Microsoft.DotNet.XHarness.CLI/Commands/GetStateCommand.cs

```diff
@@ -2,11 +2,12 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using Microsoft.DotNet.XHarness.Common.CLI.CommandArguments;
 using Microsoft.DotNet.XHarness.Common.CLI.Commands;
 
 namespace Microsoft.DotNet.XHarness.CLI.Commands
 {
-    internal abstract class GetStateCommand : XHarnessCommand
+    internal abstract class GetStateCommand<TArguments> : XHarnessCommand<TArguments> where TArguments : IXHarnessCommandArguments
```

Comment:
Is this abstract class needed since it used only for AppleGetStateCommand?

Reply:
Good catch. I inherited Android one too and deleted some code there